opmon/metadata.py

```diff
@@ -104,7 +106,7 @@ def write(self) -> None:
         view_query = f"""
             CREATE OR REPLACE VIEW `{self.project}.{self.dataset}.{view_name}` AS (
                 SELECT *
-                FROM `{self.project}.{self.dataset}_derived.{PROJECTS_TABLE}`
```

Comment:
With `tmp` being used for the preview, we'll need to remove the hardcoded `_derived` dataset assumption here since there is no `tmp_derived` dataset